cmd/check_container.go

```diff
@@ -23,6 +23,7 @@ import (
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+	b64 "encoding/base64"
```

Comment:
Since this is from the go library, there isn't a need to add an `alias` the package. It can be referenced as `base64` which is more clear. And this import can be moved to the rest of the `go` imports.